packages/react/src/runtimes/local/LocalThreadListRuntimeCore.tsx

```
@@ -3,206 +3,211 @@
+          (t) => t !== threadId,
         );
-      }
-    }
+        break;
 
-    if (lastState !== state) {
-      if (lastState === "new") {
-        this._newThread = undefined;
-      }
+      default:
+        const _exhaustiveCheck: never = lastState;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)